modules/ui/fields/address.js

```diff
@@ -288,7 +288,7 @@ export function uiFieldAddress(field, context) {
             })
             .attr('title', function(subfield) {
                 var val = tags[field.key + ':' + subfield.id];
-                return val && Array.isArray(val) && val.filter(Boolean).join('\n');
+                return (val && Array.isArray(val)) ? val.filter(Boolean).join('\n') : undefined;
```

Comment:
I fixed a minor unrelated bug here, if this condition is false, then `title="false"` is added to the DOM, which resulted in a strange tooltip:

![tooltip](https://user-images.githubusercontent.com/16009897/139615119-84a0362d-6cb5-4a97-be43-91c520a2da54.png)